lib/util.js

```diff
@@ -0,0 +1,3 @@
+const sleep = ms => new Promise((resolve, reject) => setTimeout(resolve, ms))
+
+module.exports = { wait }
```

Comment:
wait != sleep :p

Reply:
omg i cant believe i did that

Reply:
Happens to best of us, but that's the exact reason I don't use web editor :)